documentation-site/pages/getting-started/installation.mdx

```diff
@@ -11,23 +11,25 @@ export default Layout;
 
 # Installing Base UI
 
-Base UI is available on [npm](https://npmjs.com/package/baseui).
+Base UI is available on [npm](https://npmjs.com/package/baseui). This single package contains all Base UI components.
 
 ```bash
 # using yarn
-yarn add baseui styletron-react styletron-react-core styletron-standard styletron-engine-atomic
+yarn add baseui styletron-engine-atomic styletron-react
 
 # using npm
-npm install baseui styletron-react styletron-react-core styletron-standard styletron-engine-atomic
+npm install baseui styletron-engine-atomic styletron-react
 ```
 
-[Styletron](https://styletron.js.org/) is a toolkit for component-oriented styling, part of the CSS in JS family. Base UI uses
-Styletron as a peer dependency to style all the elements, so you need to add them as dependencies too.
+[Styletron](https://styletron.js.org/) is a toolkit for component-oriented styling, part of the CSS in JS family. Base UI uses Styletron as a peer dependency to style all the elements, so you need to add them as dependencies too.
```

Comment:
I believe it should be updated to styletron.org?

Reply:
Yea, I fixed all styletron links with a different PR: https://github.com/uber-web/baseui/pull/963